src/api/tasks/TaskCollaborators.js

```diff
@@ -66,6 +70,43 @@ class TaskCollaborators extends TasksBase {
         });
     }
 
+    createTaskCollaboratorsforGroup({
+        errorCallback,
+        file,
+        successCallback,
+        task,
+        group,
+    }: {
+        errorCallback: ElementsErrorCallback,
+        file: BoxItem,
+        group: { id: string },
+        successCallback: Function,
```

Comment:
We can probably type this more specifically. What is the signature and return value for `successCallback`?

Reply:
Its type is `Function` thoughout the file so I think it makes sense to keep it uniform for consistency until we can track down what the return value/signature is of `successCallback`